Lint error in pkg/v1/providers/tests/clustergen/README_ccgen.md

**Bug description**
Lint check in pre-checkin pipelines failed due to
```
pkg/v1/providers/tests/clustergen/README_ccgen.md:36 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
make: *** [Makefile:510: doc-lint] Error 1
```

**Affected product area (please put an X in all that apply)**

- ( ) APIs
- ( ) Addons
- ( ) CLI
- ( ) Docs
- ( ) IAM
- ( ) Installation
- ( ) Plugin
- ( ) Security
- (x) Test and Release
- ( ) User Experience
- ( ) Developer Experience

**Expected behavior**
Should pass

**Steps to reproduce the bug**

**Version** (include the SHA if the version is not obvious)

**Environment where the bug was observed (cloud, OS, etc)**

**Relevant Debug Output (Logs, manifests, etc)**

--- a/pkg/v1/providers/tests/clustergen/README_ccgen.md
+++ b/pkg/v1/providers/tests/clustergen/README_ccgen.md
@@ -33,7 +33,6 @@ e.g. this picks the first 10 cases using the azure infra
 export CASES=$(for i in `grep azure:v1. testdata/*.case | \
     cut -d: -f1 | uniq | cut -d/ -f7 | head -10`; do echo -n "$i "; done; echo)
 
-
 ## Details on dry-run generation of ClusterClass-based configuration
 
 Since the clustergen tests are expected to generate configuration entirely

